openlibrary/plugins/openlibrary/js/removeHighlight.js

```diff
@@ -0,0 +1,8 @@
+export default function() {
```

Comment:
These are a little small to be entirely separate files; could combine these into one file as well? Maybe `jquery_utils.js`?

- `removeHighlight.js`
- `jquery.toggleText.js`
- `jquery.fadeToggle.js`
- `jquery.toggleText.js`

Reply:
My understanding is that these are all separate libraries so mixing them one in file would be akin to putting jquery and react in the same file. It's important to me these are kept separate (and hopefully many of them can be removed)

Reply:
These 3 specifically feel more like util methods (not from any library); I think the way you've separated out the others is perfect, it's just these three :P The code is so short (and unminified, and without attribution comments) that I think these 3 functions were probably not from a library.

Reply:
anything attached to $.fn is classed as a jquery plugin that provide specific functionality. That's what I mean by a separate library. I don't mean library as in npm here.

I'd like to consider these files separately. In https://github.com/internetarchive/openlibrary/pull/1787 I attempted to remove a lot of them and I plan to try this again but more conservatively one at a time. Having these in separate files helps me do that.